modules/eco-poison-pill-operator-troubleshooting.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * /machine_management/eco-poison-pill-operator.adoc
+
+[id="troubleshooting-poison-pill-operator_{context}"]
+= Troubleshooting the Poison Pill Operator
+
+
+[id="general-troubleshooting-poison-pill-operator_{context}"]
+== General Troubleshooting 
+
+Issue::
+You want to troubleshoot issues with the Poison Pill Operator.
+
+Resolution::
+Enable the debug flag in the `poison-pill` object. The `debug` flag increases
+the verbosity of the daemons that run in the `DaemonSet` pods and run the AIDE
+checks.
+
+[id="checking-daemon-set_{context}"]
+== Checking the daemon set 
+Issue:: The Poison Pill Operator is installed but the daemon set is not available.
+
+Resolution:: Check the Operator logs for errors or warnings.
+
+[id="unsuccessful_remediation{context}"]
+== Unsuccessful remediation 
+Issue:: An unhealthy node was not remediated.
+
+Resolution:: Check and verify if the `PoisonPillRemediation` CR is created. To do so, run the following command:
++
+[source,bash,options="nowrap",role="white-space-pre"]
+----
+oc get namespace <machinehealthcheck>
+----
+
+If the `PoisonPillRemediation` CR is not created when the node turned unhealthy, check the logs of the `MachineHealthCheck` controller.
+
+If the `PoisonPillRemediation` CR is created, ensure that its name matches the unhealthy node or the machine object.
+
+[id="daemon-set-exists_{context}"]
+== Daemon set exists even after uninstalling the Poison Pill Operator 
+Issue:: The Poison Pill daemon set exists even after after uninstalling the Operator.
+
+Resolution:: To remove the Poison Pill daemon set, manually delete the `PoisonPillConfig` CR.
```

Comment:
For troubleshooting 'Unsuccessful remediation' you provide the command 'To do so, run the following command:' 
I think it would also be helpful to include the command for troubleshooting 'Daemon set exists even after uninstalling the Poison Pill Operator'.

Reply:
Thank you; added the command